src/logic/quickstart.ts

```diff
@@ -1,49 +1,341 @@
+import { exists, readFile } from '../common/io';
 import { LogCallback } from '../common/log';
+import { OutputStream } from '../common/output-stream';
+import { envVariable } from '../templates/env';
+import { installProvider } from './configure';
+import { initSuperface } from './init';
+import { detectSuperJson, installProfiles } from './install';
+import {
+  getProviders,
+  installSdk,
+  profileExists,
+  providerExists,
+} from './quickstart.utils';
 
-const execShell = promisify(exec);
-
-export async function installSdk(options?: {
+export async function interactiveInstall(options?: {
   logCb?: LogCallback;
   warnCb?: LogCallback;
+  successCb?: LogCallback;
 }): Promise<void> {
-  const result = await execShell(
-    `${
-      (await exists('yarn.lock')) ? 'yarn add' : 'npm install'
-    } @superfaceai/one-sdk`
+  let envContent = '';
+  //Super.json path
+  let superPath = await detectSuperJson(process.cwd());
+  if (superPath) {
+    //Overide existing super.json
+    if (!(await confirmPrompt('Super.json already exists.'))) {
+      options?.warnCb?.(`Super.json already exists at path "${superPath}"`);
+
+      return;
+    }
+  }
+
+  //Init SF
+  options?.successCb?.('Initializing superface directory');
+  await initSuperface(
+    './',
+    { profiles: {}, providers: {} },
+    { logCb: options?.logCb }
   );
-  if (result.stderr !== '') {
-    options?.warnCb?.(
-      `Shell command "npm install @superfaceai/one-sdk" responded with: "${result.stderr}"`
+  superPath = SUPERFACE_DIR;
+
+  //Load super.json (for checks)
+  const superJson = (
+    await SuperJson.load(joinPath(superPath, META_FILE))
+  ).unwrap();
+
+  //Ask for profile
+  const possibleProfiles = await fetchProfiles();
+  const profileResponse: {
+    profile: { scope: string; profile: string; version: string };
+  } = await inquirer.prompt({
+    name: 'profile',
+    message: 'Select a capability to install',
+    type: 'list',
+    choices: possibleProfiles.map(p => {
+      return { name: `${p.scope}/${p.profile}`, value: p };
+    }),
+  });
+  const profile = profileResponse.profile;
+
+  //Override existing profile
+  if (await profileExists(superJson, profile)) {
+    if (
+      !(await confirmPrompt(
+        `Profile "${profile.scope}/${profile.profile}" already exists.`
+      ))
+    )
+      return;
+  }
+
+  //Install profile
+  await installProfiles(
+    superPath,
+    [
+      {
+        kind: 'store',
+        profileId: `${profile.scope}/${profile.profile}`,
+        version: profile.version,
+      },
+    ],
+    {
+      logCb: options?.logCb,
+      warnCb: options?.warnCb,
+      force: true,
+    }
+  );
+
+  //Ask for providers
+  const possibleProviders = await fetchProviders(
+    `${profile.scope}/${profile.profile}@${profile.version}`
+  );
+
+  const providerResponse: { providers: string[] } = await inquirer.prompt({
+    name: 'providers',
+    message: 'Select a provider/s you want to use',
```

Comment:
Just a note: I'd rewrite this to "Select **the** provider/s you want to use" because it's more grammar friendly